### PR TITLE
deps.ffmpeg: Update libsrt to 1.5.2

### PR DESCRIPTION
**Pull request by pkviet**
### Description
This updates srt to 1.5.2. This has a commit which fixes a bug [1,2] with rendezvous mode.

[1] https://github.com/Haivision/srt/issues/2664
[2] https://github.com/Haivision/srt/commit/1cffd2f2abad1782d1218f2c2b893f26bf4bbfce

### Motivation and Context
Fix bug and keep up-to-date deps.
Companion PR in obs-studio [3].

[3] https://github.com/obsproject/obs-studio/pull/9026

### How Has This Been Tested?
Builds fine locally on wsl.

### Types of changes
- Bug fix (non-breaking change which fixes an issue) 
- Tweak (non-breaking change to improve existing functionality) 

### Checklist:
- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: deps.ffmpeg/60-srt.zsh [hunk 1]
```diff
@@ -2,9 +2,9 @@ autoload -Uz log_debug log_error log_info log_status log_output
 
 ## Dependency Information
 local name='srt'
-local version='1.5.1'
-local url='https://github.com/Haivision/srt/archive/v1.5.1.tar.gz'
-local hash="${0:a:h}/checksums/v1.5.1.tar.gz.sha256"
+local version='1.5.2'
+local url='https://github.com/Haivision/srt/archive/v1.5.2.tar.gz'
+local hash="${0:a:h}/checksums/v1.5.2-rc.2.tar.gz.sha256"
```
Comment (RytoEX) [inline deps.ffmpeg/60-srt.zsh:7]: Checksum filename needs to match.

**Review by RytoEX — CHANGES_REQUESTED** [review]

File: deps.ffmpeg/60-srt.zsh (hunk 1 above)
Comment (pkviet) [reply deps.ffmpeg/60-srt.zsh]: fixed

**Comment by pkviet** [comment]
i've fixed the mishap and retested srt to YouTube and local srt-live-transmit; worked fine.

**Review by RytoEX — CHANGES_REQUESTED** [review]
Commit message and PR title nit:
* use "libsrt" instead of "srt", in line with commit history

**Comment by pkviet** [comment]
> Commit message and PR title nit:
> 
> * use "libsrt" instead of "srt", in line with commit history

fixed.
Tested that caller / listener / rendezvous mode are all working.

**Review by RytoEX — APPROVED** [review]
Seems fine. If you've done testing, this should be fine.

**Pull request merged**